cookiecutter/repository.py

```diff
@@ -76,22 +76,27 @@ def determine_repo_dir(template, abbreviations, clone_to_dir, checkout,
     """
     template = expand_abbreviations(template, abbreviations)
 
+    repository_candidates = []
+
     if is_repo_url(template):
-        repo_dir = clone(
+        cloned_repo = clone(
             repo_url=template,
             checkout=checkout,
             clone_to_dir=clone_to_dir,
             no_input=no_input,
         )
-    else:
-        # If it's a local repo, no need to clone or copy to your
-        # cookiecutters_dir
-        repo_dir = template
+        repository_candidates.append(cloned_repo)
+
+    repository_candidates.extend([
```

Comment:
I think it would be better to have this only when ``is_repo_url`` evaluates to false.